### ISSUE TITLE
allow_private vs allow-private argument problem 0.17.0

### ISSUE THREAD

**Issue by mbrobbel** [sagiegurari/cargo-make#210]
**Describe the bug**
I'm getting the following error after upgrading to 0.17:

```
[cargo-make] INFO - cargo make 0.17.0
...
error: Found argument '--allow_private' which wasn't expected, or isn't valid in this context
	Did you mean --allow-private?
```

**To Reproduce**

```
git clone https://github.com/mbrobbel/dqcsim-rs.git
cd dqcsim-rs
make build
```

**Comment by sagiegurari** [comment]
found the issue. will fix it today. thanks!!!

**Comment by sagiegurari** [comment]
@mbrobbel just published new 0.17.1 version with the fix.
if you could also please verify it.

**Comment by mbrobbel** [comment]
0.17.1 works! Thanks.
